dmutils/email/dm_notify.py

```diff
@@ -18,11 +22,18 @@ def __init__(
             self,
             govuk_notify_api_key,
             govuk_notify_base_url='https://api.notifications.service.gov.uk',
-            logger=None
+            logger=None,
+            replacement_email_address=FROM_CURRENT_APP,
     ):
-        """Set up logging and mail client."""
+        # Set up logging and mail client.
```

Comment:
Method docstrings should be in triple quotes even if they're just a one-liner.

Reply:
I turned this from a docstring to a comment because it's not a docstring. Docstrings are for external callers to know more about the function. The information here is information the caller doesn't *really* need to know. Or at least if we get into the habit of doing this, we're going to have to start adding docstring descriptions of all methods, describing *exactly* what they do, line by line like this one does.

Reply:
Sorry, yes, you're right. I didn't pay enough attention.